### PR TITLE
added some documentation and cleanup

### PR DESCRIPTION
**Pull request by aivantg**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # '####### YOUR ANSWER BELOW ###########'

Some documentation and some cleanup! 

[//]: # '############## END ##################'

### How to Review

[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # '####### YOUR ANSWER BELOW ###########'

Just make sure i didn't break anything :P 

[//]: # '############## END ##################'

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Screenshots

[//]: # '#### YOUR SCREENSHOTS BELOW ########'
[//]: # '############## END ##################'

### Design Status

[//]: # 'If this is a frontend PR, what is the expected 
status of the UI in this PR (lo, mid, high- fi?'
[//]: # '####### LOFI/MIDFI/HIFI ###########'
[//]: # '############## END ##################'

CC: @dfangshuo

### CODE REVIEW

File: src/lib/credentials.js [hunk 1]
```diff
@@ -51,6 +52,8 @@ function getCredentials(owner) {
   return credentials;
 }
 
+// Helper functions to check whether a user has a certain credential
+
```
Comment (dfangshuo) [inline src/lib/credentials.js:56]: nit: unnecessary whitespace
```suggestion
```

Comment (aivantg) [reply src/lib/credentials.js:56]: whitespace was because i wanted to signify that the comment applied to the next group of functions, not just the next function
